tests/schemas/invalid/test_invalid_whitespaces_in_schema_text.json

```diff
@@ -0,0 +1,124 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "language": "en",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.3",
+  "survey_id": "0",
+  "session_timeout_in_seconds": 3,
+  "title": "Test Question Variants",
```

Comment:
Minor but just need to update the title and description for this schema 👍

Reply:
Title and description in test schema updated accordingly